### ISSUE TITLE
Disappearing Calendar

### ISSUE THREAD

**Issue by Milner99** [braintonik/dialogPlugIn#4]
Hi Devs,

I'm using your calendar plugin with a corona sdk project, it's working well and rendering lovely inside the app however when I tap a date the calendar disappears. Whilst this might be the default behaviour how do I stop this from happening as I'd like the calendar to be persistent on the composer scene.

Please advise

Thank you

Chris.

**Comment by ghost** [comment]
Same problem here. Did you find a solution Chris?

**Comment by Milner99** [comment]
No, the dev of the plugin is looking at something I believe. All I've done in the meantime is when the calendar event tap is fired, I re-draw the calendar straight away, the screen flickers and it's not the most elegant but it's all I can do short of removing the plugin and finding another one (or downloading the source and changing things myself but sadly we are up against a deadline and don't really have time for that)

**Comment by ghost** [comment]
Thanks for the reply. It shouldn't take 2 months to figure out this bug. If you charge for a plugin, you should fix these kind of bugs ASAP. Thanks for the tip about redrawing :)
